src/test/ast_program_test.cpp

```diff
@@ -24,6 +24,21 @@ namespace souffle {
 
 namespace test {
 
+inline std::unique_ptr<AstTranslationUnit> makeATU(std::string program = ".decl A,B,C(x:number)") {
```

Comment:
What's the point of the default argument here? Makes it slightly confusing, since it's not an obvious default.